code/game/objects/effects/contraband.dm

```diff
@@ -215,7 +215,7 @@
 /obj/structure/sign/poster/contraband/syndicate_recruitment
 	name = "Syndicate Recruitment"
 	desc = "See the galaxy! Shatter corrupt megacorporations! Join today!"
-	icon_state = "poster5"
+	icon_state = "poster_syndicate_recruitment"
```

Comment:
Keep icon_state names as numbers for now since that's the standard, however for the future it might need a doublecheck on that.

Reply:
fixed!